docs/sql-programming-guide.md

```diff
@@ -964,7 +964,7 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
     Sets the compression codec used when writing Parquet files. If either `compression` or
     `parquet.compression` is specified in the table-specific options/properties, the precedence would be
     `compression`, `parquet.compression`, `spark.sql.parquet.compression.codec`. Acceptable values include:
-    none, uncompressed, snappy, gzip, lzo, brotli, lz4, zstd.
```

Comment:
I prefer `none, uncompressed, snappy, gzip, lzo, brotli(need install ...), lz4, zstd(need install ...)`.

Reply:
Installation may not be able to solve it.

Reply:
`none, uncompressed, snappy, gzip, lzo, brotli(need install brotli-codec), lz4, zstd(since Hadoop 2.9.0)`

https://jira.apache.org/jira/browse/HADOOP-13578
https://github.com/rdblue/brotli-codec
https://jira.apache.org/jira/browse/HADOOP-13126

Reply:
got it,thanks @wangyum

Reply:
hadoop-2.9.x is officially supported in Spark?

Reply:
I think so given the download page.
![screen shot 2018-09-08 at 12 41 41 pm](https://user-images.githubusercontent.com/6477701/45250388-94d6b280-b364-11e8-85ee-1a67daa3a123.png)

Reply:
ah, ok.